### PR TITLE
14 Replace Limerick screenshots with Naas (Kildare)

### LINKED ISSUE #14: Replace Limerick screenshots with Naas (Kildare)
based on e-mail from Dan

### PR DESCRIPTION
**Pull request by Hussein-Mahfouz**
closes #14

### CODE REVIEW

File: images/newbridge_fastest.png
Comment (Robinlovelace) [inline images/newbridge_fastest.png:1]: The lines here look thinner than for the quietest network, are they all on the same scenario (Go Dutch)?

**Review by Robinlovelace — COMMENTED** [review]
Good work, quick comment from me, line widths a bit suspect?

**Comment by Hussein-Mahfouz** [comment]
Yes the fastest routes are very thin. I meant to comment that. Do you think it has something to do with the build or with the visualization? I'll look into the data on Sunday or Monday

**Comment by Hussein-Mahfouz** [comment]
Just saw your other comment - they might not all have been on go Dutch. Will redo and push edits

**Comment by Hussein-Mahfouz** [comment]
Just saw your other comment - they might not all have been on Go Dutch. Will redo and push edits

**Comment by Robinlovelace** [comment]
Sounds good.

**Comment by Hussein-Mahfouz** [comment]
I added the new screenshots for naas and newbridge:

- `naas_quietest_baseline.png`  - `naas_quietest_godutch.png`
- `naas_fastest_baseline.png`  - `naas_fastest_godutch.png`
- `newbridge_quietest_baseline.png`  - `newbridge_quietest_godutch.png`
- `newbridge_fastest_baseline.png`  - `newbridge_fastest_godutch.png`

Currently the godutch is the one being visualised, but that isn't mentioned in the paragraph above. The godutch scenario looks better with the line thickness, but the baseline scenario might make more sense if we're describing the status quo

If you want to change to visualise the baseline scenario, just replace `_godutch` with `_baseline` in the image name

**Review by Robinlovelace — APPROVED** [review]
Thanks Hussein!

**Pull request merged**